README.md

```diff
@@ -52,22 +52,25 @@ Using `flake.nix`:
   outputs = {
     self,
     nixpkgs,
+    lib,
     home-manager,
     nix-doom-emacs,
     ...
   }: {
-    nixosConfigurations.exampleHost = nixpkgs.lib.nixosSystem {
+    nixosConfigurations.exampleHost = lib.nixosSystem {
       system = "x86_64-linux";
       modules = [
         home-manager.nixosModules.home-manager
         {
-          home-manager.users.exampleUser = { pkgs, ... }: {
-            imports = [ nix-doom-emacs.hmModule ];
-            programs.doom-emacs = {
-              enable = true;
-              doomPrivateDir = ./doom.d;
-            };
-          };
+          home-manager.users.exampleUser = lib.mkMerge [
+            nix-doom-emacs.hmModule
+            { ... }: {
```

Comment:
I might be doing something wrong, but ... is this valid syntax? 

<img width="479" alt="image" src="https://user-images.githubusercontent.com/3998/162550426-7b2c8336-2baf-42a1-a2f0-5087bba87171.png">

Reply:
Okay, the whole things needs to be wrapped in `(..)`:

```nix
              ({ pkgs, ... }: {
                programs.doom-emacs = {
                  enable = true;
                  doomPrivateDir = ./config/doom.d;
                };
              })
```

However, now I get a different error:

<img width="509" alt="image" src="https://user-images.githubusercontent.com/3998/162550606-0a4a54a0-a5fc-46df-b7c0-2b1e4803a5d5.png">

Note: this is all in macOS, with `darwin-rebuild`.